app/lib/sidekiq_error_handler.rb

```diff
@@ -1,13 +1,24 @@
 # frozen_string_literal: true
 
 class SidekiqErrorHandler
+  BACKTRACE_LIMIT = 3
+
   def call(*)
     yield
   rescue Mastodon::HostValidationError
     # Do not retry
+  rescue => e
```

Comment:
For the Code Climate error "Avoid rescuing without specifying an error class.", we might be able to start `rescue`ing the following exceptions that have been recorded on my server in the last 24 hours:

- HTTP::ConnectionError
- HTTP::TimeoutError
- Mastodon::UnexpectedResponseError
- Stoplight::Error::RedLight (this might go elsewhere with the change in this PR)